sql/variables/status_variables.go

```diff
@@ -93,19 +93,13 @@ func (g *globalStatusVariables) SetGlobal(name string, val interface{}) error {
 }
 
 // IncrementGlobal implements sql.StatusVariableRegistry
-func (g *globalStatusVariables) IncrementGlobal(name string, val int) error {
+func (g *globalStatusVariables) IncrementGlobal(name string, val int) {
 	v, ok := g.varVals[name]
 	if !ok || v.Variable().GetScope() == sql.StatusVariableScope_Session {
-		return sql.ErrUnknownSystemVariable.New(name)
+		return
```

Comment:
Appropriate to panic here? Basically always means a bug right?

Reply:
The session list excludes `globalOnly`, so it's only ~297/678 of the total list. So we want to ideally not call this on global only, but noop otherwise.

Reply:
The query type counters have both scopes unfortunately, so I had to leave those for now